modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -99,5 +99,5 @@ oc get clusterserviceversion -n openshift-nmstate \
 [source, terminal]
 ----
 Name                                             Phase
-kubernetes-nmstate-operator.4.10.0-202203120157   Succeeded
+kubernetes-nmstate-operator.4.1.0-202210210157   Succeeded
```

Comment:
maybe you mean kubernetes-nmstate-operator.4.12.0-202210210157? ^ ^

Reply:
oh yes, I did mean to put 12. I think I deleted the '0'  from '10' and then realized the date also needed changing and then didn't add the 2.